src/main/java/hudson/plugins/openid/OpenIdExtension.java

```diff
@@ -127,10 +132,15 @@ public static void extendRequest(AuthRequest authRequest) throws MessageExceptio
      * @param authSuccess the authentication success.
      * @param id the identity.
      * @throws MessageException if there is a message error processing the success.
+     * @throws IllegalStateException if there is no Jenkins instance
      */
-    public static void processResponse(AuthSuccess authSuccess, Identity id) throws MessageException {
+    public static void processResponse(AuthSuccess authSuccess, Identity id) throws MessageException, IllegalStateException {
```

Comment:
There were some issues with findbugs.  This would have been a fix to get around that.